openwisp_notifications/websockets/handlers.py

```
@@ -52,23 +126,42 @@ def user_in_notification_storm(user):
+        serialized_notification = None
+        if notification_map and recipient in notification_map:
+            from openwisp_notifications.api.serializers import (
+                NotificationListSerializer,
+            )
+
+            try:
+                serialized_notification = NotificationListSerializer(
+                    notification_map[recipient]
+                ).data
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)